dictionaries/people-names/src/names.txt

```diff
@@ -3,3 +3,284 @@
 # This is not a list of individuals or famous people.
 # Please add new names to the end of the file. It will get automatically sorted
 # during the release process.
+
+Antunes
+Hawley
+Malthe
+Montel
+Nico
+Polyakovsky
+Monde
+Klimenko
+Osayd
+Pipek
+Mike
+Collerson
+Merino
+Diogo
+Gurcke
+Mauza
+Boukli
+Tregar
+Wacha
+Ryan
+Gentry
+Giedrius
+Riha
+Alejandro
+David
+Simon
+Tasnim
+Daniël
+Sparber
+Maciá
+Kelsey
+Martin
+Firas
+Luna
+Adelman
+Arkadii
+Andres
+Paulo
+Chanran
+Klintrup
+Joshua
+Kemenade
+Artem
+Dorian
+Hugh
+Isabelle
+Karin
+Kamau
+Søren
+Koki
+Mayoral
+Ondřej
+Betts
+Baptista
+Mattheeuw
+Robert
+Edison
+Akos
+Schulze
+Jerry
+Greg
+Asfour
+Schmidtke
+Jacob
+Punter
+Sylvia
+Alexander
+Härtwig
+Furton
+Edward
+Hotham
+Ramos
+Abdelkhalek
+Šedivý
+Jose
+Raychel
+Driel
+Kjær
+Robin
+Gabriel
+Affolter
+Turra
+Martinez
+Borg
+Bøvith
+Yakovets
+Argote
+Jorge
+Ellul
+Eden
+Hirose
+Siripanich
+Youhei
+Rosa
+Sylvain
+Thessel
+Fiquitiva
+Feliu
+Becker
+Miti
+Monnier
+Frazier
+Jakob
+Nomura
+Eldar
+Maurizio
+Benjamin
+Niemeijer
+Jaemin
+Tserekh
+Hugo
+Jahir
+Bernhard
+Abdu
+Romero
+Panpakorn
+Orrock
+Zsak
+Sakurai
+McKeague
+Goossens
+Douglas
+Julien
+Menan
+Lukasz
+Mierzwa
+Vincenti
+Piotr
+Cadena
+Thurau
+Sugato
+Carvalho
+Anthony
+Nate
+Broudou
+Emeršič
+Wenhaug
+Eugenio
+Joël
+Maina
+Franklin
+Henrik
+Aaron
+Kate
+Thomas
+McNamara
+Daitche
+Julian
+Denys
+Holius
+Aart
+Tommy
+Marrugo
+Fred
+Navruzov
+Hacene
+Rafferty
+Koert
+Jeremy
+Pablo
+Raphael
+Pasche
+Tewodros
+Bednárik
+Marek
+Islam
+Felix
+Borsetti
+Patrick
+Dong
+Arias
+Serhii
+Ivan
+Yatskevich
+Nataliia
+Šuppa
+Thompson
+Robles
+John
+Meshesha
+Peter
+Sofroniev
+Gordon
+Lucas
+Reinaldo
+Artem
+Navoiev
+Staniów
+Sergi
+Heikki
+Carlos
+Kamil
+Trabanco
+Sinxi
+Inggs
+Westhuizen
+Michael
+Pieter
+Juscelino
+Abellana
+Daniel
+Takeshi
+Andrei
+Harris
+Panadero
+Jung
+Rafal
+Szypulka
+Jason
+Agustín
+Nicholson
+Dima
+Lazerka
+Olivier
+Gallego
+Anders
+Jacky
+Murza
+Pavel
+Shaurya
+Christian
+Nový
+Almacellas
+Borch
+Victor
+Nicholas
+Wiedemann
+Dmytriievska
+Spagnoletti
+Papp
+Kafri
+Orsila
+Tran
+Ville
+Nishat
+Agbesi
+Picht
+Letham
+Tudor
+Albers
+Oscar
+Uppal
+Sozzi
+Santiago
+Veer
+Anandkumar
+Alexandre
+Bailey
+Chris
+Mustafayev
+Arjun
+Rocha
+Filip
+COWAN-BERGMAN
```

Comment:
@arkid15r,

I just noticed that after it had landed.

I would like to avoid hyphenated names and names in all caps. Hyphenated names are generally just two names joined with a `-`.

Reply:
Fixed in https://github.com/streetsidesoftware/cspell-dicts/pull/3292